packages/slice-machine/lib/models/common/CustomType/tab.ts

```diff
@@ -175,34 +175,35 @@ export const Tab = {
 
   organiseFields(tab: TabAsObject) {
     const tabAsArray = Tab.toArray("", tab);
-    const { fields, groups }: OrganisedFields =
-      tabAsArray.value.reduce<OrganisedFields>(
-        (
-          acc: OrganisedFields,
-          curr: { key: string; value: Field | GroupField<AsArray> }
-        ) => {
-          if (curr.value.type === sliceZoneType) {
-            return acc;
-          }
-          if (curr.value.type === FieldType.UID) {
-            return acc;
-          }
-          if (curr.value.type === FieldType.Group) {
-            return {
-              ...acc,
-              groups: [
-                ...acc.groups,
-                { key: curr.key, value: curr.value as GroupField<AsArray> },
-              ],
-            };
-          }
+    const { fields, groups }: OrganisedFields = tabAsArray.value.reduce<
```

Comment:
was this EOL created by our linter?

Reply:
Yes

Reply:
There are a few other issues picked up by prettier,
I think one of them was a weird looking type definition.
```
packages/slice-machine/lib/models/ui/LibraryState.ts
[error] packages/slice-machine/lib/models/ui/LibraryState.ts: SyntaxError: ',' expected. (7:35)
[error]   5 |   name: string;
[error]   6 |   isLocal: boolean;
[error] > 7 |   components: ReadonlyArray<[state: SliceState, store: Store]>;
[error]     |                                   ^
[error]   8 | }
[error]   9 |
```

Reply:
hmm yes I see why it complained. Some EOL seem a bit weird to me but let's see how it goes and it's not a big deal if 99% of the rest is better now